tests/integration/test_tls.py

```diff
@@ -88,7 +88,7 @@ async def test_add_tls_provider_succeeds_after_removal(ops_test: OpsTest):
             channel="edge",
             num_units=1,
             config={"generate-self-signed-certificates": "true", "ca-common-name": "zookeeper"},
-            series=SERIES,
+            series="jammy",
```

Comment:
why is this? I would argue that it is better to use constants here rather than reverting back to using strings. It would make the code more mainainable to have single point of modifications